pages_builder/pages/index.yml

```diff
@@ -15,6 +15,7 @@ content: >
         <li><a href="browse-list.html">Browse list</a></li>
         <li><a href="button.html">Buttons</a></li>
         <li><a href="contact-details.html">Contact details</a></li>
+        <li><a href="custom-dimensions.html">Custom dimensions</a></li>
```

Comment:
Is there actually any point in having this in the pages builder? - there's nothing visible to actually preview?

Reply:
I quite like having the usage example on there and seeing what tags get rendered. It's minimal at the moment but when we introduce the supplier size there'll be more examples?